build/scripts/Invoke-FormattingCheck.ps1

```diff
@@ -0,0 +1,14 @@
+
+#.SYNOPSIS
+# Checks for code formatting errors. Will throw exception if any are found.
+function Invoke-CheckBadCodeFormatting() {

```

Comment:
probably we can just unencapsulate this code and make it the toplevel contents of the script file.

Reply:
I'd prefer to leave it encapsulated, as a personal preference.